lighthouse-core/test/scripts/i18n/collect-strings-test.js

```diff
@@ -8,68 +8,65 @@
 /* eslint-env jest */
 
 const collect = require('../../../scripts/i18n/collect-strings.js');
-const esprima = require('esprima');
 
-describe('Compute Description', () => {
+function evalJustUIStrings(justUIStrings) {
+  return Function(`'use strict'; ${justUIStrings} return UIStrings;`)();
+}
+
+describe('parseUIStrings', () => {
   it('collects description', () => {
     const justUIStrings =
     `const UIStrings = {
         /** Description for Hello World. */
-        message: 'Hello World',
+        exampleString: 'Hello World',
```

Comment:
1/2 of the ` strings in this file are indented 1 too many levels, like this one.  Can we unify all these examples on 6 spaces?
```suggestion
      exampleString: 'Hello World',
```

Reply:
> 1/2 of the ` strings in this file are indented 1 too many levels, like this one. Can we unify all these examples on 6 spaces?

didn't you write these lines :P

fixed

Reply:
I don't recall ever seeing these files before in my life.